src/java/main/org/apache/zookeeper/client/StaticHostProvider.java

```diff
@@ -57,30 +61,9 @@
     public StaticHostProvider(Collection<InetSocketAddress> serverAddresses)
             throws UnknownHostException {
         for (InetSocketAddress address : serverAddresses) {
-            InetAddress ia = address.getAddress();
-            InetAddress resolvedAddresses[] = InetAddress.getAllByName((ia!=null) ? ia.getHostAddress():
-                address.getHostString());
-            for (InetAddress resolvedAddress : resolvedAddresses) {
-                // If hostName is null but the address is not, we can tell that
```

Comment:
Is this comment still relevant in the new implementation? (I don't see it in the changes)

Reply:
The comment is irrelevant because we switched to using `getHostString` which does _not_ do a host name look up and works the same whether an IP or DNS address were supplied.